nautobot/ipam/views.py

```diff
@@ -799,11 +825,15 @@ def get(self, request, *args, **kwargs):
         )
 
     def post(self, request):
+        if pks := request.POST.getlist("pk"):
+            return self.assign_ipaddresses_to_interface(pks, request)
         form = forms.IPAddressAssignForm(request.POST)
         table = None
 
         if form.is_valid():
-            addresses = self.queryset.select_related("tenant")
+            interface_id, interface_model = self._get_interface_type_and_model(request)
+            obj = interface_model.objects.get(id=interface_id)
```

Comment:
Another place that is almost identical to the blocks in 714-717 and 766-769. Can we please try to use this same code in all places?